### PR TITLE
Bug 1365983 - enter_bug.cgi should be updated to use "Data Platform and Tools"

### PR DESCRIPTION
**Pull request by Dipanks**

### CODE REVIEW

**Comment by Dipanks** [comment]
Is there something else to be done in this?

File: extensions/BMO/template/en/default/global/choose-product.html.tmpl [hunk 1]
```diff
@@ -118,7 +118,7 @@
       icon="localization.png"
     %]
     [% INCLUDE easyproduct
-      name="Cloud Services"
+      name="Data Pipeline and Tools"
```
Comment (dylanwh) [inline extensions/BMO/template/en/default/global/choose-product.html.tmpl]: I think the product is "Data Platform and Tools"

**Comment by Dipanks** [comment]
@floatingatoll  I pushed this commit with the commit message. Is this okay?

@dylanwh I made the changes.

**Comment by floatingatoll** [comment]
Yep, thanks! (re message)

**Pull request merged**